scripts/deployToken.js

```diff
@@ -0,0 +1,19 @@
+const ERC20Token = artifacts.require('ERC20Token')
```

Comment:
After installing everything with `npm install`, the following error arises when using this file: `Error: Could not find artifacts for ERC20Token from any sources`.

Reply:
Must be due to the compiler issue you had which in result didn't create the ERC20Token artifact.

Reply:
Yes, I could make it run now. BTW I think a `npx aragon contracts compile` should be before the deployToken command in `start.sh`.

Reply:
Good call. I'll do a `npx truffle compile` as `aragon contracts compile` returns en exit code of 0 even though the compiling failed.